### PR TITLE
Fix -409 Adding local image selection for background Image in Options…

### LINKED ISSUE #409: Adding local image selection for background Image in Options.html
**I'm submitting a ...**  (check one with "x")
- [x] feature request

**Current behavior:**

Currently,  we can only save URL  for background image for chrome bot.

**Expected behavior:**
Can upload image and use it as background.

**Steps to reproduce:**

**Would you like to work on the issue?**
Yes  I am already working on it.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #409 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

Here is the Working.
ezgif com-video-to-gif 8

### CODE REVIEW

**Comment by sk9331657** [comment]
@gabru-md  @stealthanthrax  Please approve it.

**Comment by sk9331657** [comment]
@Akshat-Jain @gabru-md  Please merge it

File: src/js/options.js [hunk 1]
```diff
@@ -111,3 +113,13 @@ restore.addEventListener("click", ()=>{
...
+
+
+
+localimage.addEventListener("change",()=>{
+    
+    var tmppath = URL.createObjectURL(localimage.files[0]);
+    localStorage.setItem("theValue",tmppath);
+    alert("Image Updated");
+  
+});
```
Comment (Akshat-Jain) [inline src/js/options.js]: Add EOF. Thanks :-)

Comment (sk9331657) [reply src/js/options.js]: @Akshat-Jain  What I have to write to mention EOF??

Comment (sk9331657) [reply src/js/options.js]: Done

Comment (sk9331657) [reply src/js/options.js]: @gabru-md  Please merge it.

**Comment by sk9331657** [comment]
@gabru-md  Please Merge it

File: src/js/options.js (hunk 1 above)
Comment (ms10398) [reply src/js/options.js]: Can you add an empty line before EOF

Comment (sk9331657) [reply src/js/options.js]: Ok

**Comment by sk9331657** [comment]
@Akshat-Jain  Added EOF :). Please merge it .

**Pull request merged**